env/dev/load-secrets.tpl

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Maybe rename the script?   I initially thought that a script that "loads secrets" would *read* secrets from SecretsManager.  It's not clear that it's a destructive/overwriting update to SecretsManager.